package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "polls",
+  "description": "Polls app for nextcloud",
+  "version": "0.9.0",
+  "author": "Rene Gieling",
+  "license": "AGPL-3.0",
+  "private": true,
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack --progress --hot --config js-src/webpack.config.js --watch",
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules --config js-src/webpack.config.js",
+    "test": "cross-env NODE_ENV=development webpack --progress --hide-modules --config js-src/webpack.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nextcloud/polls.git"
+  },
+  "bugs": {
+    "url": "https://github.com/nextcloud/polls/issues"
+  },
+  "homepage": "https://github.com/nextcloud/polls#readme",
+  "dependencies": {
+    "vue": "^2.5.16",
+    "axios": "^0.17.1",
+    "moment": "^2.22.1",
+    "lodash": "^4.17.10"
+  },
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "not ie <= 8"
+  ],
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-stage-3": "^6.24.1",
+    "cross-env": "^5.1.3",
+    "css-loader": "^0.28.8",
+    "file-loader": "^1.1.6",
+    "vue-loader": "^13.7.0",
+    "vue-template-compiler": "^2.5.16",
+    "webpack": "^3.6.0"
```

Comment:
Any reason you not use the latest 4.x version of we pack? :wink:

Reply:
Yes. I don‘t like it. 😄 
Seriously, it is because, I am totally new to it. So any review is strong welcome. I will look after it and update.

Reply:
Hmmm... 
No I get " webpack.optimize.UglifyJsPlugin has been removed, please use config.optimization.minimize instead."

Note to myself: Don't change a running system. 😈

Reply:
Maybe https://github.com/nextcloud/deck/pull/459/files will help you a bit, as I did the same version upgrade last week. ;)

Reply:
Hmmm... To much to understand for me right now between coffee and bread. I reverted the commit and will make the update in another PR. Or maybe someone else can repair the configuration.

BTW: I get the Error "Error: webpack.optimize.UglifyJsPlugin has been removed, please use config.optimization.minimize instead".